greek/use_the_rope/grab_the_ax/grab_the_ax

```diff
@@ -0,0 +1,4 @@
+Αρπάζεις το τσεκούρι το οποίο βρίσκεται κάτω απο το παράθυρο σε θήκη για περίπτωση φωτιάς και κόβεις το σχοινί.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.